.github/workflow_scripts/env_setup.sh

```diff
@@ -1,5 +1,5 @@
 function setup_build_env {
-    python -m pip install --upgrade pip
+    python -m pip install pip==21.3.1 # Capping pip to prevent tensorrt install failure
```

Comment:
Looks like the tesorrt installation error is gone. Do we still need this change? Would it affect other packages?

Reply:
We do not need it at the moment.
Ideally it should not affect other packages because it's just pip.
We will treat this PR as a draft PR in case we see this failure in the future.
(As this failure was experienced by a lot of open-source teams outside AutoGluon at some point - in the shared links)